src/main/java/nl/tudelft/contextproject/gui/MenuController.java

```diff
@@ -42,15 +45,13 @@
     @FXML private void initialize() {
         final String name = ContextTFP.getScript().getName();
 
-        if (name.equals("") || name == null) {
-            setLabel("None");
+        if (name.equals("")) {
+            setScriptLabel("None");
         } else {
-            setLabel(name);
+            setScriptLabel(name);
         }
-
-        btnCreateScript.setOnAction(event -> {
-            CreateScriptController.show();
-        });
+        
+        setVersionLabel("0.6");
 
         btnLoadScript.setOnAction(event -> {
             FileChooser fileChooser = new FileChooser();
```

Comment:
This initialize method is still a bit on the long side and it could be shortened by initialize the load button in a separate method.